docs/integrations/bigquery.mdx

```diff
@@ -11,7 +11,16 @@ Supports Bigquery since version 0.3.4
 
 ### BigQuery connection profile
 
-The profiles.yml file to connect to BigQuery should contain a profile in the following format:
+After installing Elementary's dbt package upon running `dbt deps`,
+you can generate Elementary's profile for usage with `edr` by running the following command within your project:
+
+```shell
+dbt run-operation elementary.generate_elementary_cli_profile
+```
+
```

Comment:
1. Explain what will happen:
"The command will print to terminal..."

2. Explain the next step:
"copy the output to [HOME]/.dbt/profiles.yml, if it doe's not exist, here is a command to create it..."

3. What are the fields that are missing?